<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../MainWindow.qrc">
    <normaloff>:/Res/Res/Mario.ico</normaloff>:/Res/Res/Mario.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>视角</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_Up">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/Res/Res/btn_Up.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_Down">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/Res/Res/btn_Down.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_Left">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/Res/Res/btn_Left.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="btn_Right">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/Res/Res/btn_Right.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="btn_Forward">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/Res/Res/btn_Forward.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="btn_Back">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/Res/Res/btn_Back.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>渲染</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_Filter_I">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/Res/Res/CloudRender.png);
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>133</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="PCLQVTKWidget" name="qvtkWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>50</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>属性：</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="textBro_1">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>8888</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="title">
         <string>参数：</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="textBro_2">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>8888</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1339</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PCLQVTKWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">pcl\visualization\qvtk_compatibility.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_Up</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btn_Up_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Down</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btn_Down_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Forward</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btn_Forward_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Back</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btn_Back_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btn_Left_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btn_Right_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Filter_I</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btn_Filter_I_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_Up_clicked()</slot>
  <slot>btn_Down_clicked()</slot>
  <slot>btn_Forward_clicked()</slot>
  <slot>btn_Back_clicked()</slot>
  <slot>btn_Left_clicked()</slot>
  <slot>btn_Right_clicked()</slot>
  <slot>btn_Filter_I_clicked()</slot>
 </slots>
</ui>
